roles/pulibrary.oracle.java8/molecule/default/molecule.yml

```diff
@@ -0,0 +1,20 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: instance
+    image: centos:7
```

Comment:
Please see the above question regarding CentOS

Reply:
the way the repo is set up, we never really use this. we rely `molecule/default/molecule.yml` from the repo root.